### ISSUE TITLE
See lots of errors with the newest release

### ISSUE THREAD

**Issue by Redding43** [springfall2008/batpred#456]
Firstly I should say I am not ruling out it being a Octopus or GivTCP issue as my OctoAid app is erroring when trying to load rates currently

however, started seeing these errors after the update of Predbat today

![IMG_3224](https://github.com/springfall2008/batpred/assets/122637671/fbed083e-17e4-42e2-9f88-8117b9f6e411)
![IMG_3223](https://github.com/springfall2008/batpred/assets/122637671/64b3c2b9-303a-4f93-bf0e-6cdaa4d8ffd4)
![IMG_3225](https://github.com/springfall2008/batpred/assets/122637671/ccb5bb6c-20fc-4be8-9e71-2c9d7458b4cb)

**Comment by springfall2008** [comment]
It looks like REST isn’t working, maybe try restarting GivTCP ?

**Comment by Redding43** [comment]
Tried a restart. Seeing if it has fixed it. Spotted this just now in GivTCP logs though

![IMG_3226](https://github.com/springfall2008/batpred/assets/122637671/5d6d019b-828b-44e9-8627-43f7e701fd70)

**Comment by gcoan** [comment]
@Redding43 I've seen something similar happen.  Restarting GivTCP in my case fixed it.

I did log #330 to see if any enhancements could be made to predbat's retry mechanism.

Also BTW I did add a couple of sample automations to the documentation https://springfall2008.github.io/batpred/output-data/ that monitor predbat and GivTCP for disconnects to give a better warning when one of these components stops working
